migration-engine/core/src/commands/apply_migrations.rs

```diff
@@ -40,9 +40,14 @@ impl<'a> MigrationCommand for ApplyMigrationsCommand {
         let applier = connector.database_migration_step_applier();
         let migration_persistence = connector.new_migration_persistence();
 
+        migration_persistence.initialize().await?;
```

Comment:
Is this command the only one that requires an initialized migration persistence?

Reply:
Yes, it's on purpose, and the tests validate that. We want to have a single place to initialize the migration persistence, because we want to handle the case where the persistence is not initialized, and the database is not empty separately (the user will need to manually baseline their migrations history). I'll document this in the code, thanks for pointing that out.